pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -204,6 +305,9 @@ type Controller struct {
 	nodeLister          corelisters.NodeLister
 	nodeInformerSynced  cache.InformerSynced
 
+	//TODO: remove this after UT fixes
+	getPodsAssignedToNode func(nodeName string) ([]*v1.Pod, error)
```

Comment:
Remove line 308-309?

Reply:
all UT has fixed to not reference to this function. so removed it as well.